packages/build/webpack/webpack.dev.config.js

```diff
@@ -29,16 +29,19 @@ module.exports = {
     filename: '[name].js',
     chunkFilename: '[name].js',
   },
-  devtool: false,
+  devtool: 'source-map',
```

Comment:
Should we use source-map in development? [The docs](https://webpack.js.org/configuration/devtool/) say it's a setting recommended for production builds. Did you consider using `eval-source-map` or any other setting?

Reply:
I'm happy to use `eval-source-map`